src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -67,8 +68,16 @@ spec:
 
 No matter if you are using Helm Charts or GitOps and Kustomize, just put the ServiceMonitor CR next to your app and apply it in the same way. Once it's applied you can check either the _ServiceMonitors Overview_ or _ServiceMonitors Details_ dashboards, or just search for the new metrics ingested in your installation's Grafana to make sure that your containers are being scraped by the new monitoring agents.
 
-__Warning:__ The ServiceMonitor needs to be labeled with `application.giantswarm.io/team: <YOUR-TEAM-NAME>` for the Prometheus Agent to be able to discover it and start collecting metrics.
+__Warning:__ The ServiceMonitor needs to be labeled with `observability.giantswarm.io/tenant: <YOUR-TENANT-NAME>` for the metrics agent to be able to discover it and start collecting metrics. Also, please make sure that the tenant you're setting with the label does actually exist in one of the [Grafana organization CRs](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/creating-grafana-organization/).
```

Comment:
```suggestion
__Warning:__ The ServiceMonitor needs to be labeled with `observability.giantswarm.io/tenant: <YOUR-TENANT-NAME>` for the metrics agent to be able to discover it and start collecting metrics. Also, you need to make sure that the tenant you're setting with the label does actually exist in one of the [Grafana organization CRs](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/creating-grafana-organization/). Any metrics that are sent to a tenant that is not referenced by at least one Grafana Organisation is not ingested by Mimir! 
```

We need to be more precise in the consequences of not setting an existing tenant.